eno-core/src/main/java/fr/insee/eno/core/model/question/PairwiseQuestion.java

```diff
@@ -25,7 +25,6 @@
 public class PairwiseQuestion extends SingleResponseQuestion {
 
     Response response = null;
```

Comment:
Indeed...

`PairwiseQuestion` inherits `SingleResponseQuestion` since it is represented in a `QuestionItem` in DDI. Question items are mapped in a list of single response questions in the Eno questionnaire object.

Making PairwiseQuestion not inherit SingleResponseQuestion would imply making a selection on the mapping expression of single response questions in the questionnaire object + adding a special filed for the pairwise question in the questionnaire.

The other reason is that some properties are mapped using the super class annotation.

We could sacrifice the second reason to respect the Liskov principle, but the first reason make me feel it should remain like this (note: the DDI modelling of the pairwise question might be changed to a `QuestionGrid`, this would fix this issue).

Reply:
📑 I should wright javadoc on this field meanwhile

Reply:
_Or:_ remove the Response mapping on the SingleResponseQuestion class an copy it in every single response question object except pairwise